client/src/components/Troubleshooting/Troubleshooting.jsx

```diff
@@ -1,42 +1,379 @@
+              handleOnTroubleshootingChange("draintype", e.target.value);
+            }}
+          >
+            <option value="">Select a Drain Type</option>
+            <option value="PCN">{draintype}</option>
+          </select>
+          {troubleshooting.draintype && (
+            <div className="q2">
+              <h2>What type of issue are you having?</h2>
+              <select
+                name="issue"
+                className="troubleshooting-dropdown"
+                value={troubleshooting.issue}
+                onChange={(e) => {
+                  handleOnTroubleshootingChange("issue", e.target.value);
+                }}
+              >
+                <option value="">Select an Issue</option>
+                <option value="volumeChange">
+                  Change in the amount of output
+                </option>
+                <option value="quality">Change in the quality of output</option>
+              </select>
             </div>
-        </div>
+          )}
+          {troubleshooting.issue === "volumeChange" && (
+            <div className="q3">
+              <h2>How has it changed?</h2>
+              <select
+                name="changeInVolume"
+                className="troubleshooting-dropdown"
+                value={troubleshooting.changeInVolume}
+                onChange={(e) => {
+                  handleOnTroubleshootingChange(
+                    "changeInVolume",
+                    e.target.value
+                  );
+                }}
+              >
+                <option value="">Select a Change</option>
+                <option value="increase">Sudden increase</option>
+                <option value="decrease">Sudden decrease</option>
+              </select>
+            </div>
+          )}
+          {troubleshooting.changeInVolume === "increase" && (
+            <div className="q4">
+              <h2>
+                Have you had persistent increased output for more than 24-48h?
+              </h2>
+              <select
+                name="persistentIncrease"
+                className="troubleshooting-dropdown"
+                value={troubleshooting.persistentIncrease}
+                onChange={(e) => {
+                  handleOnTroubleshootingChange(
+                    "persistentIncrease",
+                    e.target.value
+                  );
+                }}
+              >
+                <option value="">Select a Change</option>
+                <option value="yes">Yes</option>
+                <option value="no">No</option>
+              </select>
+            </div>
+          )}
+          {troubleshooting.persistentIncrease === "no" && (
+            <div className="q6">
+              <h2>Any changes in your health? Fever/pain?</h2>
+              <select
+                name="persistentincreasechange"
+                className="troubleshooting-dropdown"
+                value={troubleshooting.persistentincreasechange}
```

Comment:
Nit try to be consistent with your naming. I see you are using camel case through out most of it so hopefully we stay consistent on this?

Reply:
Okay nice catch thank you!